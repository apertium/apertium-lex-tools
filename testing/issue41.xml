<rules>
<rule weight="2.0">
<match lemma="William" tags="np.*"><select lemma="Guillem" tags="np.*"/></match>
<match tags="det.def.*"/>
<or>
  <match case="Aa" tags="n.*"/>
  <match case="Aa" tags="adj.*"/>
</or>
</rule>
</rules>
