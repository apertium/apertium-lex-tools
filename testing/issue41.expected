^William<np><ant><m><sg>/Guillem<np><ant><m><sg>$ ^the<det><def><sp>/el<det><def><GD><ND>$ ^Bastard<n><sg>/Bastardo<n><GD><sg>$^,<cm>/,<cm>$ ^William<np><ant><m><sg>/Guillem<np><ant><m><sg>/William<np><ant><m><sg>$ ^the<det><def><sp>/el<det><def><GD><ND>$ ^bastard<n><sg>/bastardo<n><GD><sg>$ 
